core/internal/tensorboard/tbproto/tfevent.proto

```diff
@@ -38,6 +38,19 @@ message TFEvent {
 //
 // We only include fields that are relevant to us.
 message Summary {
+  message Image {
```

Comment:
why some in the event proto and some in a different file proto file?

Reply:
The protos are structured the same way as in TensorBoard. Technically I could define Histogram inside Summary or Image outside of Summary, but keeping it similar makes it easier to cross-reference to the TB source.

Reply:
Ah that makes sense, let’s keep it this way! Thanks